packages/@react-spectrum/dialog/stories/Dialog.stories.tsx

```diff
@@ -49,34 +49,10 @@ storiesOf('Dialog', module)
     'with hero, isDimissable',
     () => renderHero({isDismissable: true, onDismiss: action('dismissed')})
   )
-  .add(
```

Comment:
can we not specify these anymore? (S/M/L)

Reply:
I'll readd